### ISSUE TITLE
How to make a constant?

### ISSUE THREAD

**Issue by satarics** [jdryg/dls-schematics#29]
How to create a constant if possible?

**Comment by Azzydon** [comment]
For 0 or 1 you can pull an input to VCC or GND. For larger binary numbers you can tie the input of a latch to a binary pattern and read the latch when you need the constant. You could use an array of diodes and a multiplexer to create a simple rom that provides different values depending on the mux selector inputs. Or you could use some kind of rom that you load with a pattern that works for you. 

https://electronics.stackexchange.com/questions/505304/understanding-diode-rom

**Comment by jdryg** [comment]
> How to create a constant if possible?

Right-click on an input port and select Constant. As far as I remember both gate-level and transistor-level circuits support constant input ports.
